src/natcap/invest/urban_nature_access.py

```diff
@@ -520,21 +556,84 @@ def execute(args):
             population_alignment_task,
         ])
 
-    aggregate_admin_units_task = graph.add_task(
+    # only do split population if there are population group fields in the
+    # admin units vector
```

Comment:
Somewhat nit-picky but it seems like we've been moving away from `admin` to `aoi`. Would it make sense for this to be "in the aoi vector"?

Reply:
Makes sense to keep it consistent!  I've updated this to reflect both possibilities in 032cf92b9.